src/LiquidityPool.sol

```diff
@@ -90,6 +90,10 @@ contract LiquidityPool is Auth, IERC7540 {
         emit File(what, data);
     }
 
+    function recoverTokens(address to, uint256 amount) external auth {
```

Comment:
We should add asset as another param. Both for non liquidity pool contracts we add it to, but even for liquidity pool, someone can accidentally send USDC to the FRAX liquidity pool

Reply:
Great point. I'll update it